### ISSUE TITLE
EU-Subsidies: bootstrap the datapackage pipeline

### ISSUE THREAD

**Issue by cyberbikepunk** [openspending/openspending#1093]
Implement the validation of the tabular data-packages coming from the sourcing step.

**Comment by akariv** [comment]
@cyberbikepunk could you please elaborate a little on this task?

**Comment by cyberbikepunk** [comment]
@akariv I renamed the task because _bootstrapping_ seemed like a much better description of what I'm try to accomplish here and because the schema validation of the source data is implemented as a pipeline processor. The objective of the bootstrap code is to:

- [ ] detect all folder corresponding to a country or region (NUTS code)
- [ ] convert and enhance the hand written YAML description file into a JSON datapackage 
- [ ] drop empty extractor processor modules (e.g. scraper) in the local directory.

Basically the job of this module is to update all the pipelines.  I sandboxed the code yesterday on the plane. Will push it for you to look at it.
